### DISCUSSION TITLE
时光胶囊页面底下的github部分内容

### DISCUSSION THREAD

**Discussion by aofly** [imsyy/home#255] [Q&A] [answered]
### 问题描述

![image](https://github.com/imsyy/home/assets/129035806/c6df4b3f-d9bb-4aca-b25d-8f7ec5b4bf5e)
请问这部分内容可以修改吗？可以的话在哪里修改呢

**Comment by aonoa** [comment, marked as answer]
src/components/GithubProject.vue

**Comment by Jerry-FaGe** [comment]
![image](https://github.com/imsyy/home/assets/40235071/f792d2c3-b688-4e77-a8f3-1b2515d5e62a)
但是名字或仓库过长的时候会换行导致下面的简介第二行只露个头
